components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.css

```diff
@@ -110,7 +110,6 @@
 }
 .xdialog-box .question {
```

Comment:
@sdumitriu +1, however I had a discussion with @allasm about removing thaat block and about how the yes/no buttons are too small and difficult to click, and we agreed on setting a `min-width` for buttons (as a better version of the `width: 100px` rule. I suggest something in the lines of `5em'. Wdyt?